.github/workflows/build.yml

```diff
@@ -90,7 +90,7 @@ jobs:
       - name: Setup .NET Core
         uses: actions/setup-dotnet@v1
         with:
-          dotnet-version: 6.0.100
+          dotnet-version: 7.0.100
```

Comment:
I think it's better to use 7.0.x for feature bug and security fixes

Reply:
I think so https://github.com/actions/setup-dotnet#supported-version-syntax

Reply:
Nice :)